s2i/pom.xml

```diff
@@ -80,21 +80,36 @@
         <version>${maven.exec.plugin.version}</version>
         <executions>
           <execution>
-            <id>exec</id>
+            <id>syndesis-builder-image-generator</id>
             <phase>package</phase>
             <goals>
               <goal>exec</goal>
             </goals>
+            <configuration>
+              <executable>java</executable>
+              <arguments>
+                <argument>-jar</argument>
+                <argument>${project.build.directory}/syndesis-builder-image-generator.jar</argument>
+                <argument>--to=${basedir}/src/main/docker/project</argument>
+              </arguments>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-local-repo</id>
+            <phase>package</phase>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+            <configuration>
+              <executable>bash</executable>
+              <arguments>
+                <argument>${basedir}/copy_mvn_repo.sh </argument>
+                <argument>${syndesis.rest.version}</argument>
```

Comment:
Why is there a specific "rest" version ? Should be always 'syndesis.version' (one of the benefits of a monorepo)

Reply:
just reusing what was defined previously.